### ISSUE TITLE
RNA-seq Pipeline needs index_bam ResourceRequirement

### ISSUE THREAD

**Issue by jasonwalker80** [genome/analysis-workflows#585]
Using Cromwell, a recent build failed at index_bam with no memory resource requirement. Should Cromwell have a default value if it's not defined?

Either way, I've edited the processing profile to get a succeeded build:
```
    - class: ResourceRequirement
      ramMin: 6000
      tmpdirMin: 25000
```

https://github.com/genome/analysis-workflows/blob/4b5970dcf08bac2598e62ffd2673e67a25004c40/definitions/tools/index_bam.cwl#L11

**Comment by apaul7** [comment]
facing a similar issue with the older `Germline Exome Alignment, QC, and GATK-HC`(a820b19a60ba49d5ba60acdfa4428d67) processing profile. Is it best to just update each of the tools in the processing profile to have a minimum memory requirement?

**Comment by jasonwalker80** [comment]
I think it's the most reasonable thing to do at this point. As @tmooney had pointed out, toil had a default of 2GB of RAM. At a minimum defining that as the ramMin should be equivalent. I went ahead and defined a larger resource requirement to be safe. As long as the changes seem idempotent. For example, certain multi-thread operations might result in unordered output. Changing the ramMin for our LSF scheduler should be fine.

**Comment by jasonwalker80** [comment]
Closed by #589
